### PR TITLE
Feature/adc/retour utilisateur

### PR DESCRIPTION
**Pull request by AmauryDanisCousandier**
tadam we have the time displayed in the song row

### CODE REVIEW

File: assets/populate.py [hunk 1]
```diff
@@ -20,16 +21,16 @@ def getOrCreateAlbum(name, artistId):
 	return out["id"]
 
 def getOrCreateGenre(names):
-    ids = []
```
Comment (Arthi-chaud) [inline assets/populate.py:23]: oops :)

File: assets/populate.py [hunk 2]
```diff
@@ -58,6 +61,11 @@ def populateFile(path, midi, mxl):
 	})
 	print(res.json())
```
Comment (Arthi-chaud) [inline assets/populate.py:63]: Is this function invoked anywhere?

File: assets/populate.py (hunk 1 above)
Comment (AmauryDanisCousandier) [reply assets/populate.py:23]: tbh i didnt code that line

Comment (AmauryDanisCousandier) [reply assets/populate.py:23]: i just changed space indent to tab indent

**Review by Arthi-chaud — CHANGES_REQUESTED** [review]
Great! A few things to do next (additionally to the comment)
- [ ] Could the `DurationComponent` be moved to its own file
- [ ] So that we could have the duration in the `SongLobbyView` (next to the song's level)

**Pull request merged**